frame/support/src/lib.rs

```diff
@@ -1550,17 +1554,17 @@ pub mod pallet_prelude {
 ///
 /// ```
 /// #[frame_support::pallet]
-/// // NOTE: Example is name of the pallet, it will be used as unique identifier for storage
+/// // NOTE: The name of the pallet will be provided by construct_runtime, and will be used as
+/// // unique identifier for storage. It is not defined in the pallet itself.
```

Comment:
```suggestion
/// // NOTE: The name of the pallet is provided by `construct_runtime` and is used as
/// // the unique identifier for the pallet's storage. It is not defined in the pallet itself.
```
cannot make suggestions on top of suggestions :shrug: